pymeasure/instruments/temptronic/temptronic_base.py

```diff
@@ -790,29 +790,35 @@ def error_status(self):
     def cycling_stopped(self):
         """:returns: ``True`` if cycling has stopped.
         """
-        return self.temperature_condition_status_code == TemperatureStatusCode.CYCLING_STOPPED
+        status = (self.temperature_condition_status_code & (0b1 << 5))
+        return status == TemperatureStatusCode.CYCLING_STOPPED
```

Comment:
The beautiful thing about IntFlags is that they support the `in` operator, so you can do more idiomatically
```
TemperatureStatusCode.CYCLING_STOPPED in self.temperature_condition_status_code
```

Reply:
That reads much more nicely. Done!